src/PeriodEnd/LevyPayments/DeployLevyPayments/sql/ddl/PeriodEnd.Transient.LevyPayments.ddl.views.sql

```diff
@@ -18,10 +18,10 @@ AS
 SELECT
 	acc.AccountId,
 	acc.AccountName,
-	acc.Balance - ISNULL(stat.LevySpent,0) [Balance]
+	(acc.Balance - ISNULL(stat.LevySpent,0) - ISNULL(ta.Amount,0)) [Balance]
 FROM Reference.DasAccounts acc
-LEFT JOIN LevyPayments.AccountProcessStatus stat
-	ON acc.AccountId = stat.AccountId
+LEFT JOIN LevyPayments.AccountProcessStatus stat ON acc.AccountId = stat.AccountId
+LEFT JOIN TransferPayments.vw_TransferedAmountForAccount ta ON acc.AccountId = ta.FundingAccountId
 WHERE acc.AccountId IN (SELECT AccountId FROM Reference.DasCommitments)
```

Comment:
This is a bugbear of mine... but can you put the ON clause on a new line? If not and there is also an AND then the ON gets lost.